smarts/core/tests/test_observations.py

```diff
@@ -134,16 +138,19 @@ def sample_vehicle_pos(lens, rgb, ogm, drivable_area, vehicle_pos):
 
     # Check if vehicles are rendered at the expected position
     # RGB
+    tolerance = 2
```

Comment:
@Gamenot is it fine to have a small tolerance here when checking rgb, ogm and drivable_grid_map? The motorbike model width is smaller than other vehicles so location isn't as accurate

Reply:
Just note that this tolerance growth formula is (2n+1)^2.  (i.e. 1, 9, 25, 49, 81, ...)